### PR TITLE
VTCCA-3765: test coverage

### PR DESCRIPTION
**Pull request by PeterPerhac**

### CODE REVIEW

File: app/controllers/propertyLinking/ClaimProperty.scala [hunk 1]
```diff
@@ -135,7 +135,7 @@ class ClaimProperty @Inject()(
         declaration: CapacityDeclaration,
         uarn: Long,
         address: String,
-        clientDetails: Option[ClientDetails] = None)(implicit request: AuthenticatedRequest[_]): Future[Unit] =
```
Comment (darrell-plant) [inline app/controllers/propertyLinking/ClaimProperty.scala:138]: 👍

File: app/uk/gov/hmrc/propertylinking/controllers/valuations/ValuationsController.scala [hunk 2]
```diff
@@ -63,63 +64,49 @@ class ValuationsController @Inject()(
...
+              assessmentsWithLinks(assessments, submissionId, owner),
+              backlink,
+              assessments.address,
+              assessments.capacity),
+            owner
+          ))
+
+      pLink
+        .flatMap {
+          case Some(EmptyAssessments()) | None => Future.successful(notFound)
```
Comment (darrell-plant) [inline app/uk/gov/hmrc/propertylinking/controllers/valuations/ValuationsController.scala:94]: 🍭

**Comment by darrell-plant** [comment]
Well done. LGTDz 👍

**Comment by habeeb01** [comment]
Nice one Pete - 👍  LGTM

**Pull request merged**